src/aap_eda/core/models/rulebook.py

```diff
@@ -36,11 +37,26 @@ class Meta:
     name = models.TextField(null=False)
     description = models.TextField(null=True, default="")
     # TODO: this field should not have a default value.
+    # TODO: should the content of this field be validated?
+    # https://issues.redhat.com/browse/AAP-19202
     rulesets = models.TextField(null=False, default="")
     project = models.ForeignKey("Project", on_delete=models.CASCADE, null=True)
     created_at = models.DateTimeField(auto_now_add=True, null=False)
     modified_at = models.DateTimeField(auto_now=True, null=False)
 
+    # For instrospection purposes we need to return a dict
+    def get_rulesets_data(self) -> list:
```

Comment:
@Alex-Izquierdo Should this be list[dict] instead of list?

Reply:
That's correct. Updated.